### PR TITLE
Improve the ingestion volume reduction guide

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Describe the global configuration option at the agent level.

### Preview

https://docs-staging.datadoghq.com/antoine.dussault/external_ingestion_guide/tracing/guide/trace_ingestion_volume_control/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/guide/trace_ingestion_volume_control.md [hunk 1]
```diff
@@ -42,62 +38,86 @@ Use the ingestion monthly usage KPI to get an estimation of your usage compared
...
+|--------------------|-------------------|-----------------------|---------|
+| `auto`             | Agent             | the Datadog Agent distributes sampling rates to tracing libraries    | 10 traces per second per Agent |
+| `rule`             | Tracing Libraries | Libraries' defined sampling percentage for specific services.   | null                 |
+
+
+Some additional ingestion reasons, surfaced in the ingestion control page, and as a tag of `datadog.estimated_usage.apm.ingested_bytes` metric may be responsible for your ingestion volume:
+
+| ingestion reason   | Where             | Ingestion Mechanism Description | Default |
+|--------------------|-------------------|-----------------------|---------|
+| `error`            | Agent             | Sampling of errors uncaught by the head-based sampling             | 10 traces per second per Agent (null if rules are defined) |
```
Comment (maycmlee) [inline content/en/tracing/guide/trace_ingestion_volume_control.md:99]:
```suggestion
| `error`            | Agent             | Sampling of errors uncaught by the head-based sampling.             | 10 traces per second per Agent (null, if rules are defined) |
```

File: content/en/tracing/guide/trace_ingestion_volume_control.md [hunk 2]
```diff
@@ -42,62 +38,86 @@ Use the ingestion monthly usage KPI to get an estimation of your usage compared
...
+| `rare`            | Agent             |  Sampling of rare traces (catching all combinations of a set span tags)        | 5 traces per second per Agent (null if rules are defined) |
```
Comment (maycmlee) [inline content/en/tracing/guide/trace_ingestion_volume_control.md:100]:
```suggestion
| `rare`            | Agent             |  Sampling of rare traces (catching all combinations of a set of span tags).        | 5 traces per second per Agent (null, if rules are defined) |
```

File: content/en/tracing/guide/trace_ingestion_volume_control.md [hunk 3]
```diff
@@ -42,62 +38,86 @@ Use the ingestion monthly usage KPI to get an estimation of your usage compared
...
+| `manual`             | In-code         | In-code decision override to keep/drop a span and its children    | null |
+| `analytics`          | Agent and Tracing Libraries | Deprecated ingestion mechanism that samples single spans without the full trace   | null                 |
```
Comment (maycmlee) [inline content/en/tracing/guide/trace_ingestion_volume_control.md:102]:
```suggestion
| `analytics`          | Agent and Tracing Libraries | Deprecated ingestion mechanism that samples single spans without the full trace.   | null                 |
```

File: content/en/tracing/guide/trace_ingestion_volume_control.md [hunk 4]
```diff
@@ -42,62 +38,86 @@ Use the ingestion monthly usage KPI to get an estimation of your usage compared
...
+
+Additionally, other products can be responsible for sampled span volume:
+
+- `synthetics` & `synthetics-browser`: API and browser tests are connected to the trace generated by the test.
```
Comment (maycmlee) [inline content/en/tracing/guide/trace_ingestion_volume_control.md:106]:
```suggestion
- `synthetics` and `synthetics-browser`: API and browser tests are connected to the trace generated by the test.
```

File: content/en/tracing/guide/trace_ingestion_volume_control.md [hunk 5]
```diff
@@ -42,62 +38,86 @@ Use the ingestion monthly usage KPI to get an estimation of your usage compared
...
+- `rum`: Requests from web and mobile applications are linked to the corresponding backend traces.
+- `lambda` & `xray`: Traces generated from AWS lambda functions instrumented with X-Ray or Datadog libraries.
```
Comment (maycmlee) [inline content/en/tracing/guide/trace_ingestion_volume_control.md:108]:
```suggestion
- `lambda` and `xray`: Traces generated from AWS lambda functions instrumented with X-Ray or Datadog libraries.
```

File: content/en/tracing/guide/trace_ingestion_volume_control.md [hunk 6]
```diff
@@ -42,62 +38,86 @@ Use the ingestion monthly usage KPI to get an estimation of your usage compared
...
 
+Read more about ingestion reasons in the [ingestion mechanisms][2] documentation
```
Comment (maycmlee) [inline content/en/tracing/guide/trace_ingestion_volume_control.md:110]:
```suggestion
Read more about ingestion reasons in the [Ingestion Mechanisms documentation][2].
```

**Comment by maycmlee** [comment]
@dussault-antoine should I go ahead and merge this?

**Review by maycmlee — APPROVED** [review]

**Comment by dussault-antoine** [comment]
> @dussault-antoine should I go ahead and merge this?

Let's do it, thanks !

**Pull request merged**